### ISSUE TITLE
Uknown production Norway is renewable, not fossil

### ISSUE THREAD

**Issue by Gazer75** [electricitymaps/electricitymaps-contrib#4699]
I've notice data listed as "other renewable" from ENTSOE is put in the unknown category with an estimated 700g emission. This is far from the truth. It should probably be in its own category.

**Comment by VIKTORVAV99** [comment]
700g co2eq/kWh is the Electricity Maps default for unknown data.
In Norway's case we can probably set it to 230g co2eq/kWh as the data from ENTSO-E says it's "Other renewable", this carbon intensity is the same as biomass and is the highest co2 intensity of renewable sources. But without any more detailed breakdown I think that is the best we can do for now.

@pierresegonne is this something we can do without a more detailed breakdown?

**Comment by pierresegonne** [comment]
If we have a good source that explains that only biomass is in unknown for NO then we can override the unknown  emission factor :)

**Comment by VIKTORVAV99** [comment]
> If we have a good source that explains that only biomass is in unknown for NO then we can override the unknown emission factor :)

Currently all the unknown values come from the ENTSO-E section Other renewable so it should be either: wind, solar, hydro, biomass or geothermal. But that is the best we got at the moment.

**Comment by large** [comment]
I agree on the biomass uncertainty is best to choose 700 gCO2eq/kWh, since it is unknown :)

But may I ask why Norway have hydro on 24 gCO2eq/kWh
![image](https://user-images.githubusercontent.com/535446/207455993-478179af-bec3-44fb-b673-44def0c46b6f.png)

While the rest of europe have 11 gCO2eq/kWh ? 
![image](https://user-images.githubusercontent.com/535446/207456504-99839264-f985-48f7-8dbd-05730788281a.png)

I noticed the source are different IPCC 2014 and Sweden (rest of europe) have UNECE 2022.
Are Norway polluting the double with their hydro than rest of europe?

**Comment by Gazer75** [comment]
> Currently all the unknown values come from the ENTSO-E section Other renewable so it should be either: wind, solar, hydro, biomass or geothermal. But that is the best we got at the moment.

Thing is saying biomass is 700g CO2/kWh is kind of crazy. It's fine for places that actually have coal as a power source.
Worst case this is biomass for district heating or pulp from logging mills that sometimes have a local power plant.
Seems a number below 300 for biomass is what I've found after some searching.

Basically the "other renewable" should use the worst value for all countries which I guess is biomass? Not the current 700 which is from coal.

> Are Norway polluting the double with their hydro than rest of europe?

I think the emission to build the massive hydro complexes is quite high. There are hundreds of kilometers of tunnels and many big dams that have been built. And in recent years they have been rebuilding several dams as the old ones were not safe anymore.

**Comment by VIKTORVAV99** [comment]
> My opinion is to make each "category" from Entso-E (B01 to B20) as "statics" so you actually can compare with your neighbour.
> If one country has hydro = 20g and another has hydro = 11g you can not compare them 1-to-1
There are some issues with this as not all parsers return as detailed breakdowns as ENTSO-E does as well as it would clutter down the UI if we had 20(+) categories.

There have however been some discussions about using dynamic emission factors for the categories by using static figures for the underlying data. I.e. 50% run of the river + 50% reservoir would be different from 100% reservoir unlike now (these would vary based on the hourly output). See https://github.com/electricitymaps/electricitymaps-contrib/discussions/4219#discussioncomment-3110126 for more info.

But right now there is a priority to find more regional emissions and improve our estimation models.

As for the Norwegian co2intensity this can definitely be adjusted if we can verify the source/methodology.

**Comment by pierresegonne** [comment]
I just got confirmation, the UNECE report which we used for hydro emission factors only accounts for EU-27 countries, so that excludes Norway. 

If you find a good (comprehensive, independent and authoritative) data source for the lifecycle emission factors for hydro in Norway, please share and/or create a PR to update the zone config :)

**Comment by Gazer75** [comment]
> I am Norwegian and I found this on the NVE: https://www.nve.no/nytt-fra-nve/nyheter-energi/lavt-klimagassutslipp-knyttet-til-norsk-stromforbruk-i-2020/ stating that Norway has as low as 8 gCO2eq/kWh in 2020.
> 
> Their "live stats" is living here: https://www.nve.no/energi/energisystem/kraftproduksjon/hvor-kommer-strommen-fra/ (BI rapport in bottom)

Problem here is that NVE set the CO2e/kWh to 0 for hydro or any renewable. This is far from the truth as they do not add in construction emissions.

**Comment by large** [comment]
> I just got confirmation, the UNECE report which we used for hydro emission factors only accounts for EU-27 countries, so that excludes Norway.

Norway only follows EU rules :) Norwegians "wanted" it that way...

> If you find a good (comprehensive, independent and authoritative) data source for the lifecycle emission factors for hydro in Norway, please share and/or create a PR to update the zone config :)

NVE is trusted as the government regulator of all power sources in Norway. I find them trustworthy, but I guess you guys don't :)
Not going to use any more time on this, I just wanted electricitymaps to be comparable.

@Gazer75 if that is true then the 11 gCO2e/kWh for rest of Europe isn't correct then? They have build the hydrosites too...
Norway compared with Sweden almost equal skills, building time/force and landscape.

When searching for this theme I found a site that use statics for comparison
https://www.nowtricity.com/country/norway/
https://www.nowtricity.com/country/sweden/
In the [faq ](https://www.nowtricity.com/faq/)the stated using the [ipcc ](https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf#page=7)from 2018 as constants

**Comment by VIKTORVAV99** [comment]
It's not that NVE is not trustworthy in it self but it's hard to verify any of these claims without a methodology about how they calculated them and what base values they used.

This is also a problem for Sweden with Vattenfalls EPD on nuclear emissions so it's not something that is specific towards Norway in any way.

For now we only use sources that can be peer reviewed (and it's very likely it will stay that way) for transparency sake and if there is source for Norwegian hydro then we will use the IPCC values for it like we do now. (The same IPCC values as you linked I might add).

We all want to have the most accurate emission factors for each zone but that don't mean we should compromise our methodology to do so.

**Comment by large** [comment]
> It's not that NVE is not trustworthy in it self but it's hard to verify any of these claims without a methodology about how they calculated them and what base values they used.

Well, NVE try to be as transparent as possible in the description. Their "mistake" is that is on Norwegian 😁

> For now we only use sources that can be peer reviewed (and it's very likely it will stay that way) for transparency sake and if there is source for Norwegian hydro then we will use the IPCC values for it like we do now. (The same IPCC values as you linked I might add).

For me it looses credibility when different categories are weightet different in each country. I would rater see an audited constants list with all known sources and use that all over. That way it would be easy (and right) to compare countries side-by-side.

> We all want to have the most accurate emission factors for each zone but that don't mean we should compromise our methodology to do so.

I totally agree about accuracy. Maybe your method should have a kind of optional "normalize" matrix as alternative?

**Comment by Gazer75** [comment]
> For me it looses credibility when different categories are weightet different in each country.

Thing is this will never be the same as countries may have different composition of types of power stations.
Some have small footprint river stations with a lot of water flowing through, which would probably have the best CO2e/kWh.
Others can be massive complexes with huge dams and many km of tunnels that produced a lot of CO2 to construct.
The best is probably the shortest tunnel and highest hydraulic head you can do to get the most energy out.

**Comment by VIKTORVAV99** [comment]
> > For me it looses credibility when different categories are weightet different in each country.
> 
> Thing is this will never be the same as countries may have different composition of types of power stations. Some have small footprint river stations with a lot of water flowing through, which would probably have the best CO2e/kWh. Others can be massive complexes with huge dams and many km of tunnels that produced a lot of CO2 to construct. The best is probably the shortest tunnel and highest hydraulic head you can do to get the most energy out.

An even easier example might be with solar as even if all panels have the same construction costs (in terms of co2) and install cost they will generate different amounts of electricity depending on where they are placed in the world affecting their resulting life-cycle emissions. (This is due to solar irradiance and daylight hours if anyone want to take a deep dive into it.)

This makes it more or less an requirement to have different values for each zone if we want to be as accurate as possible and the same goes for hydro, wind, etc. These are not static figuress that can be compared 1 to 1 between countries.

**Comment by VIKTORVAV99** [comment]
I tried to use https://ec.europa.eu/eurostat/web/energy/data/energy-balances to match the other data to generation values from Eurostat but things are not adding up...

ENTSOE-E is reporting most other as other renewable right now but I can't determine where said other comes from. It's not solar as it's also reported during night and biomass and waste should be reported separately (and is for most bidding zones).

It's possible it's still biomass or renewable waste as not all subzones report all the values but there seems to be now way for us to determine this based on the current data we have.

**Comment by Gazer75** [comment]
Coal is not a source in Norway at all unless you include Svalbard ;)
The only time gas or diesel would be used is for peak load on district heating.

**Comment by VIKTORVAV99** [comment]
We are closing down some longstanding issues that have't had any activity in a while. If you believe this is still an issue let us know and we can reopen it.
